.github/workflows/tests.yml

```diff
@@ -124,7 +124,6 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v5
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
for opens source projects AKA public repos, this is not supported anymore

Reply:
but is it safe? does this mean anybody can upload to our coverage?

Reply:
we do not have alternatives, it's the only way. it is safe cause you need access to the gh repo to post the coverage

Reply:
alright, can you rephrase your commit message to something more "official"? 😅

Reply:
done